### PR TITLE
All pull requests for 1.1.6 combined

### PR DESCRIPTION
**Pull request by allasm**
@sashaandjic - when all 1.1.6 requests are merged need a quick smoke test for all the issues included

### CODE REVIEW

**Comment by sashaandjic** [comment]
TESTED

**Comment by sashaandjic** [comment]
tested

File: matching-notification-api/src/main/java/org/phenotips/matchingnotification/finder/MatchFinder.java [hunk 1]
```diff
@@ -36,11 +36,37 @@
...
-     * Find matches for a given patient.
+     * Returns a name which can be used to select a specific matcher to be run.
+     * @return a name
+     */
+    String getName();
+
+    /**
+     * Finds matches for a given patient.
+     *
+     * TODO: this should be a default method, once we switch to Java 1.8
```
Comment (sdumitriu) [inline matching-notification-api/src/main/java/org/phenotips/matchingnotification/finder/MatchFinder.java]: PhenomeCentral is using 1.8 already.

Comment (allasm) [reply matching-notification-api/src/main/java/org/phenotips/matchingnotification/finder/MatchFinder.java]: Isn't Phenotips 1.3.x using 1.7?

Comment (sdumitriu) [reply matching-notification-api/src/main/java/org/phenotips/matchingnotification/finder/MatchFinder.java]: Yes, but this is not main PhenoTips.

**Pull request merged**